samples/dashboards/demos/climate/demo.css

```diff
@@ -267,6 +267,10 @@
     overflow: auto;
 }
 
+#selection-grid .hc-dg-outer-container {
+    height: 330px !important;
```

Comment:
Can we avoid using `!important`?  Maybe the design of the DataGrid is wrong, to set inner style on the element?

Reply:
The size is set by dataGridComponent. maybe we should add some margin there?

Reply:
Maybe we should set this value from the JS?